CHANGELOG.md

```diff
@@ -17,13 +17,21 @@ The REST API versions in the table are just for your information as the values a
 
 ## 7.0.0
 
+[Migration guide](https://sdk.gooddata.com/gooddata-ui/docs/migration_guide_7.html) from version 6.3.2
+
 ### Added
 
-- The source code structure is formatted and validated by [Prettier](https://prettier.io/) (FET-130)
+- Added Combo chart ([doc](https://sdk.gooddata.com/gooddata-ui/docs/combo_chart_component.html))
+- Pivot table subtotals can be managed by a submenu
+- The source code structure is formatted and validated by [Prettier](https://prettier.io/)
 
 ### Changed
 
-- Pivot table prop `groupRows` is true by default, thus grouping is enabled by default (BB-1385)
+- Library is built using Typescript 3.3.4000
+- Pivot table prop `groupRows` is true by default, thus grouping is enabled by default
+- Pivot table applies escaping to prevent xss
+- Pivot table subtotals styling and fixes
+- Legend icons of line and area chart are shown as circles instead of squares
 
```

Comment:
- The library now uses Typescript 3.3.4000.
- The pivot table property `groupRows` is `true` by default, which means that grouping is now enabled by default.
- The Pivot Table component applies escaping to prevent XSS.
- Styling updates and various fixes have been applied to subtotals in the Pivot Table component.
- The Legend icons of the line chart component and the area chart components are shown as circles instead of squares.